### DISCUSSION TITLE
On ChromeOS

### DISCUSSION THREAD

**Discussion by UnBrewsual** [jdholtz/auto-southwest-check-in#323] [General]
I just finished setting this neat tool up on ChromeOS's Linux. I built a bash to prompt for the 3 things and execute. So far so good!

Thanks for building this tool

**Comment by jdholtz** [comment]
Thanks! Glad to hear you are finding it helpful.

**Comment by UnBrewsual** [comment]
@jdholtz heya, with southwests changes, this tool dead now? I am traveling again and tried to fire it up, but a new error is appearing: NoSuchElementException, it appears to be on line 203 of the webdriver.py, maybe? I increased the timeout to 20 seconds, but it didn't help.

**Reply by jdholtz** [reply]
No, I'm working on migrating to the new API in #380, so it is not dead!

**Reply by UnBrewsual** [reply]
Ah great to hear, looks like my timing is lining up with the fix. giggity
